website/docs/components/tabs/index.js

```diff
@@ -6,23 +6,7 @@
 import Component from '@glimmer/component';
 import { action } from '@ember/object';
 
-import { COLORS } from '@hashicorp/design-system-components/components/hds/tag';
-
 export default class Index extends Component {
-  get STATES() {
-    // these are used only for presentation purpose in the showcase
-    return ['default', 'hover', 'active', 'focus'];
-  }
-
-  get COLORS() {
-    return COLORS;
-  }
-
-  @action
-  toggleHighlight() {
-    this.showHighlight = !this.showHighlight;
-  }
```

Comment:
We don't still need these?

Reply:
The toggleHighlight action is used in the separate Showcase and is still working there. This one isn't being used in the Web site.

I couldn't find where the colors were being used for anything.